src/Providers/BlamableServiceProvider.php

```diff
@@ -49,4 +55,15 @@ private function registerBlamableFieldMacros()
             }
         });
     }
+
+    public function register()
+    {
+        parent::register();
+
+        if (! defined('ASTALPAERT_BLAMABLE_ROOT')) {
```

Comment:
I think you could replace this with

`$this->mergeConfigFrom(__DIR__ . '/../config/blamable.php', 'blamable');`